src/pages/Profile.tsx

```diff
@@ -22,11 +22,21 @@ const Profile: React.FC = () => {
 
 	const email = user?.email ?? ''
 
+	// TODO: add the motto to the database
+	const motto = 'Člověk nežije proto, aby pracoval, ale pracuje, aby žil.'
+
+	const photo = userData?.picture?.large ?? ''
```

Comment:
This is a huge problem. :sweat_smile: After Registration refactoring/errors were merged (now on master), userData file was DELETED. Profile nickname is also set the minute you register and e-mail is also taken from our own db so there was no reason to use them anymore.

We don't want to fetch data from random API when we already have our own database, **I understood this issue as giving users avatar field in Firebase firestore database and fetching it from our own data**.

Either way, your avatar branch can't be approved and merged when it uses data we no longer have. I'd recommend rebasing your branch on master and going from there.

Reply:
Ok, so i have to adjust it. I didn't realize the random user would be gone.